Controller/Api/CommentController.php

```diff
@@ -95,7 +95,7 @@ public function deleteCommentAction($id)
         try {
             $this->commentManager->delete($comment);
         } catch (\Exception $e) {
-            return \FOS\RestBundle\View\View::create(array('error' => $e->getMessage()), 400);
+            return View::create(array('error' => $e->getMessage()), 400);
```

Comment:
The use statement is for `FOS\RestBundle\Controller\Annotations\View`, which is an annotation. That's probably not what you want here.